test/Microsoft.TemplateEngine.Core.UnitTests/CombinedStreamTests.cs

```diff
@@ -20,7 +20,7 @@ public class CombinedStreamTests
         [InlineData(1024 * 1024 + 10000)]
         public void CanReadStream(int requestedCount)
         {
-            Random rnd = new Random();
+            var rnd = new Utils.CryptoRandom();
```

Comment:
😕 Why did this need to change?

Reply:
Though it's just a test code - and hence didn't necessarily require the change - I as well believe there is no harm in flipping this in test code as well and hence avoiding any small risk of possible copy-paste spreading back to production code or any need to review security static analysis scan results.

I'm not strongly opinionated though - let me know if you see an issue with this change in test code.

Reply:
Based on @bekir-ozturk comment about increasing public surface of our API I removed the usage from tests